src/main/telemetry/srxl.c

```diff
@@ -296,8 +296,8 @@ bool srxlFrameGpsLoc(sbuf_t *dst, timeUs_t currentTimeUs)
     UNUSED(currentTimeUs);
     gpsCoordinateDDDMMmmmm_t coordinate;
     uint32_t latitudeBcd, longitudeBcd, altitudeLo;
-    uint16_t altitudeLoBcd, groundCourseBcd, hdop;
-    uint8_t hdopBcd, gpsFlags;
+    uint16_t altitudeLoBcd, groundCourseBcd, hpop;
```

Comment:
```suggestion
    uint16_t altitudeLoBcd, groundCourseBcd, pdop;
```

Reply:
Gosh thank you for noticing!  Strange that it compiled with that error?  Fixed.